RasterPropMonitor/Core/RPMCEvaluators.cs

```diff
@@ -2332,6 +2332,36 @@ internal VariableEvaluator GetEvaluator(string input, out bool cacheable)
                 case "FUNDS":
                     return (string variable, RPMVesselComputer comp) => { return Funding.Instance != null ? Funding.Instance.Funds : 0.0; };
 
+                // CommNet
+                case "COMMNETCONNECTED":
+                    return (string variable, RPMVesselComputer comp) => {
+                        return ((vessel.connection != null) && (vessel.connection.IsConnected)) ? 1.0 : 0.0;
+                    };
+                case "COMMNETSIGNALSTRENGTH":
+                    return (string variable, RPMVesselComputer comp) => {
+                        return (vessel.connection != null) ? (float)vessel.connection.SignalStrength : 0.0;
+                    };
+                case "COMMNETVESSELCONTROLSTATE":
+                    return (string variable, RPMVesselComputer comp) => {
+                        if (vessel.connection == null)
+                            return 0.0;
+                        
+                        switch (vessel.connection.ControlState)
+                        {
+                            case CommNet.VesselControlState.None: return 0.0;
+                            case CommNet.VesselControlState.Probe: return 2.0;
```

Comment:
What I meant was, why even numbers?  Why not 1, 2, 3, 4, 5, etc?  It's okay as-is, I was curious why you were using only the even numbers for return values.